model/src/cfd.rs

```diff
@@ -1848,6 +1848,9 @@ pub fn calculate_profit(payout: Amount, margin: Amount) -> (SignedAmount, Percen
 }
 
 /// Compute the liquidation price for the party going long.
+///
+/// The formulas used do not take into account the fees that can flow between the two parties. This
```

Comment:
🙃 Not sure if the first sentence adds much value. I'd rather formulate in a way saying _what it does_, e.g. the second sentence.

Reply:
Actually, the added paragraph is kind of wrong because it assumes that the leverage passed cannot be the effective leverage, which is not true. We don't (cannot) enforce that the `leverage` argument is the initial leverage!